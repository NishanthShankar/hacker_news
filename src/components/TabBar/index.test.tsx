import React from 'react'
import { BrowserRouter } from 'react-router-dom'
import Enzyme, { shallow } from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
import Component from './index'
import toJson from 'enzyme-to-json'
Enzyme.configure({ adapter: new Adapter() })

const Element = (): JSX.Element => (
  <BrowserRouter>
    <Component />
  </BrowserRouter>
)

describe('<TabBar />', () => {
  it('renders without props', () => {
    const tree = shallow(<Element />)
    expect(toJson(tree)).toMatchSnapshot()
  })
})
