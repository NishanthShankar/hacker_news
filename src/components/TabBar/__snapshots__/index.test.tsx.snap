// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabBar /> renders without props 1`] = `
<BrowserRouter>
  <TabBar />
</BrowserRouter>
`;
